### ISSUE TITLE
Magma /query ::count should work for collections

### ISSUE THREAD

**Issue by coleshaw** [mountetna/monoetna#854]

**Comment by coleshaw** [comment]
* Should also update the Timur Map "count" feature to use this API update
* Update wiki documentation (if available)

**Comment by coleshaw** [comment]
@graft , can you provide an example query that you would like to work with `::count`?

**Comment by graft** [comment]
Yeah I think perhaps we wrote this down wrong, and this is not a problem with ::count but ::has, which doesn't evaluate correctly for collections.
